main.js

```diff
@@ -0,0 +1,102 @@
+let companies = [
+  { img: "http://placehold.it/140x100", name: "Walmart", address: "10000 Harbor Santa Ana, CA", review: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure" },
+  { img: "http://placehold.it/140x100", name: "Target", address: "1000 Seal Beach Blvd. Seal Beach, CA", review: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure" },
+  { img: "http://placehold.it/140x100", name: "7-11", address: "100 Barranca Irvine, CA", review: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure" }
+];
+
+// The DOM_ sign preceding a variable name is a naming convention that represents a DOM element.
+// DOM_form represents the user search form.
+// DOM_businesses represents the users search results that will be displayed on the page.
+// DOM_term represents the users search value.
+let DOM_form = document.getElementById('form-search');
+let DOM_businesses = document.getElementById('businesses');
+let DOM_term = DOM_form.querySelector('input');
+
+// Listen for `submit` events on the DOM_form in the DOM.
+DOM_form.addEventListener('submit', (event) => {
+
+  // Cancels the default behavior for submit events.
```

Comment:
Remove all comments explaining the details of your implementation. Rely on meaningful names. Only explain things that might confuse a professional developer.